.npmignore

```diff
@@ -0,0 +1,4 @@
+.eslintrc
```

Comment:
Tests, including linting, need to be able to work when installed (ie, `npm explore foo && npm install && npm test`).
```suggestion
```